IsraelHiking.Web/package.json

```diff
@@ -54,7 +54,7 @@
     "intl": "1.2.5",
     "invert-color": "2.0.0",
     "isomorphic-xml2js": "^0.1.3",
-    "jszip": "^3.6.0",
+    "jszip": "^3.7.1",
```

Comment:
Not exactly what I would've expected - I would've expected to see here two lines with the relevant libraries (`resample` and `moving-media`) with their relevant versions...
Is updating the version of the jszip library is relevant to this commit?